src/Request/CreditRequest.php

```diff
@@ -0,0 +1,17 @@
+<?php
+
+namespace Nexy\PayboxDirect\Request;
+
+/**
+ * @author Sullivan Senechal <soullivaneuh@gmail.com>
+ */
+class CreditRequest extends AbstractBearerTransactionRequest
+{
+    /**
+     * {@inheritdoc}
+     */
+    public function getRequestId()
```

Comment:
I would call this `getRequestType`… id sound unique, but you can make several `CreditRequest`, so…